src/pages/templates/pages/partial/dashboard_panel_popular_reads_guest_messages.html

```diff
@@ -0,0 +1,23 @@
+{% if data.view == 'assigned' %}
+    A list of recently assigned readings displays for students with a
+    Clusive account connected with a teacher.
+{% elif data.view == 'recent' %}
+    {% if data.all|length == 0 %}
+        You have no recent readings yet. Go to the 
+        <a href="{% url 'library_style_redirect' view='period' %}">library</a>
```

Comment:
Should probably be view='public' for guests, since they have no period.

Reply:
Right.